site/guide/fairings.md

```diff
@@ -27,7 +27,7 @@ provides a rich set of mechanisms such as [request guards] and [data guards]
 that can be used to solve problems in a clean, composable, and robust manner.
 
 As a general rule of thumb, only _globally applicable_ actions should be
-effected through fairings. You should _not_ use a fairing to implement
+affected through fairings. You should _not_ use a fairing to implement
```

Comment:
I'm fairly certain [effected is correct here](http://www.diffen.com/difference/Affected_vs_Effected).

Reply:
Hmm... Oh boy, yup. You're right. I guess I've never seen the word "effected" actually used correctly.

Reply:
Can you discard this change? I'll merge the typo fix above thereafter.